torch/distributed/__init__.py

```diff
@@ -6,9 +6,9 @@ def is_available():
     """
     Returns ``True`` if the distributed package is available. Otherwise,
     ``torch.distributed`` does not expose any other APIs. Currently,
-    ``torch.distributed`` is available on Linux and MacOS. Set
+    ``torch.distributed`` is available on Linux, MacOS and Windows. Set
     ``USE_DISTRIBUTED=1`` to enable it when building PyTorch from source.
-    Currently, the default value is ``USE_DISTRIBUTED=1`` for Linux and
+    Currently, the default value is ``USE_DISTRIBUTED=1`` for Linux and Windows,
```

Comment:
I wonder if we should make ``USE_DISTRIBUTED=0`` by default, and let users set it to 0 when necessary?